src/js/components/StepNav/StepNav.stories.js

```diff
@@ -0,0 +1,74 @@
+import React from "react";
+import { storiesOf } from "@storybook/react";
+import StepNav from "./StepNav";
+
+const stories = storiesOf("StepNav", module);
+
+stories.add("Default", () => (
+  <div style={{ padding: 20 + "px" }}>
```

Comment:
don't know if we still need the padding still with the latest changes from Bill and Sara.

Reply:
It's still a little tight even with those paddings. :\

Reply:
<img width="1231" alt="Screen Shot 2019-06-05 at 10 20 02 AM" src="https://user-images.githubusercontent.com/1455979/58963637-8c889800-877b-11e9-820c-6b73a519b78e.png">